design-system-docs/src/_component_examples/_targeted_content/adviser.erb

```diff
@@ -1,5 +1,4 @@
 ---
-title: Adviser
```

Comment:
I'm slightly confused about why these don't have titles and the grid examples do. Is it because the grid ones are using iframes?

Reply:
Oh good catch, I don't think the grid ones need titles anymore either.

Reply:
Oh wait I worked it out. If you don't add a `title` it uses the slug as the title. I removed these because the titles were just repeating the slug. It's probably clearer though to always have them.